qiskit_ibm/ibm_backend.py

```diff
@@ -769,9 +748,6 @@ def run(    # type: ignore[override]
             circuits: An individual or a
                 list of :class:`~qiskit.circuits.QuantumCircuit` or
                 :class:`~qiskit.pulse.Schedule` objects to run on the backend.
-                A :class:`~qiskit.qobj.QasmQobj` or a
```

Comment:
The docstring for this `run()` and the one for `IBMRetiredBackend.run()` should be updated to not say `Qobj`